ros_ethercat_model/include/ros_ethercat_model/hardware_interface.hpp

```diff
@@ -95,12 +96,20 @@ class ActuatorCommand
 public:
   ActuatorCommand() :
     enable_(0),
-    effort_(0)
+    effort_(0),
+    effort_command_type_(EffortCommandType::UNSPECIFIED)
   {
   }
 
   bool enable_;  //!< Enable this actuator
-  double effort_;  //!< Force to apply (in Nm)
+  double effort_;  //!< Effor to apply, unit depends on effort_command_type_ field
```

Comment:
Typo in the comment "effor"

Reply:
Fixed in fa9de9e929c50a35484106d74a707e081cd06e32.